app/decorators/course_decorator.rb

```diff
@@ -111,6 +111,20 @@ def funding_option
     end
   end
 
+  def main_address
```

Comment:
This does already exist on the provider...

Reply:
While this should exist on the provider, the method that currently exists in the provider has the wrong formatting